OSRMTextInstructions.podspec

```diff
@@ -0,0 +1,47 @@
+Pod::Spec.new do |s|
+
+  # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.name = "OSRMTextInstructions.swift"
```

Comment:
There are a lot of ways to spell this project, aren’t there?

* osrm-text-instructions.swift (repository name)
* OSRMTextInstructions.swift (pod name)
* Project OSRM text instructions (pod summary)
* OSRM Text Instructions (pod description)
* `OSRMTextInstructions` (framework/module name)

Maybe we should simplify things a little bit by taking “.swift” out of the pod name, since this is the only Swift-compatible OSRM Text Instructions library.